### ISSUE TITLE
"use strict"; gives error: "Expected an assignment or function call and instead saw an expression"

### ISSUE THREAD

**Issue by bluesmoon** [eslint/eslint#1185]
I have a line inside a function that looks like this:

``` javascript
    "use strict";
```

eslint complains about

```
Expected an assignment or function call and instead saw an expression  no-unused-expressions
```
